docker-compose.yml

```diff
@@ -0,0 +1,118 @@
+# docker-compose for Socorro development.
+#
+# Note: Requires docker-comopse 1.10+.
+version: "2"
+services:
+  # Processor app
+  processor:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile.processor
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile.processor
+    env_file:
+      - docker/config/docker_common.env
+      - docker/config/processor.env
+    depends_on:
+      # - statsd -- FIXME(willkg): broken for some reason
+      - localstack-s3
+      - postgresql
+      - elasticsearch
+      - rabbitmq
+    volumes:
+      - .:/app
+
+  # Webapp base
+  webapp-base:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile.webapp
+    volumes:
+      - .:/app
+
+  # Webapp app
+  webapp:
+    extends:
+      service: webapp-base
+    env_file:
+      - docker/config/docker_common.env
+      - docker/config/webapp.env
+    depends_on:
+      # - statsd -- FIXME(willkg): broken for some reason
+      - localstack-s3
+      - postgresql
+      - elasticsearch
+      - rabbitmq
+    command: ["--dev"]
```

Comment:
Based on this I'm assuming that you're setting ENTRYPOINT in the Dockerfile. This is good.